payments-core/src/main/java/com/stripe/android/networking/RequestHeadersFactory.kt

```diff
@@ -1,11 +1,13 @@
 package com.stripe.android.networking
 
+import androidx.annotation.RestrictTo
 import com.stripe.android.ApiVersion
 import com.stripe.android.AppInfo
 import com.stripe.android.Stripe
 import java.util.Locale
 
-internal sealed class RequestHeadersFactory {
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+sealed class RequestHeadersFactory {
```

Comment:
Same as above re: auditing what we're making public now

Reply:
no longer needed